src/App.js

```diff
@@ -1,25 +1,80 @@
 import React from 'react';
+import { Clock } from './components/Clock';
 
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
-};
+export class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 0,
+  };
+
+  componentDidUpdate() {
+    const showClock = document.getElementById('showClock');
+    const hideClock = document.getElementById('hideClock');
+
+    if (this.state.isClockVisible === false) {
+      showClock.hidden = false;
+      hideClock.hidden = true;
+    }
+
+    if (this.state.isClockVisible === true) {
+      showClock.hidden = true;
+      hideClock.hidden = false;
+    }
```

Comment:
```suggestion
    if (this.state.isClockVisible === false) {
      showClock.hidden = false;
      hideClock.hidden = true;
    } else {
      showClock.hidden = true;
      hideClock.hidden = false;
    }
```
Такое обычно через if..else пишется